src/modules/peek/Peek.UI/Extensions/WindowExtensions.cs

```diff
@@ -20,10 +20,19 @@ public static Size GetMonitorSize(this Window window)
             MONITORINFO info = new ();
             info.cbSize = 40;
             PInvoke.GetMonitorInfo(hwndDesktop, ref info);
-            var monitorWidth = info.rcMonitor.left + info.rcMonitor.right;
-            var monitorHeight = info.rcMonitor.bottom + info.rcMonitor.top;
+            double monitorWidth = info.rcMonitor.left + info.rcMonitor.right;
+            double monitorHeight = info.rcMonitor.bottom + info.rcMonitor.top;
 
             return new Size(monitorWidth, monitorHeight);
         }
+
+        public static double GetMonitorScale(this Window window)
+        {
+            var hwnd = new HWND(window.GetWindowHandle());
+            var dpi = PInvoke.GetDpiForWindow(new HWND(hwnd));
+            double scalingFactor = dpi / 96d;
```

Comment:
Is it possible for the baseline dpi to not be 96 for some monitors? Seems like you can also use this api alternatively to avoid dealing with DPI at all: https://learn.microsoft.com/en-us/windows/win32/api/shellscalingapi/nf-shellscalingapi-getscalefactorformonitor